R/initialize_modules.R

```diff
@@ -74,45 +74,48 @@ initialize_module <- function(parameters, data, module_name) {
 
   if (module_class_name == "Fleet") {
     module_fields <- setdiff(module_fields, c(
-      "log_expected_index",
-      "proportion_catch_numbers_at_age"
+      "log_index_expected",
+      "log_landings_expected",
+      "composition_numbers_at_age",
+      "proportion_numbers_at_age",
+      "observed_index_in_weight",
+      "observed_landings_in_weight"
     ))
 
     fleet_types <- get_data(data) |>
       dplyr::filter(name == module_name) |>
       dplyr::pull(type) |>
       unique()
 
-    if ("landings" %in% fleet_types) {
-      module_fields <- setdiff(module_fields, c(
-        "log_q",
-        "random_q",
-        "estimate_q"
-      ))
-    } else {
-      module_fields <- setdiff(module_fields, c(
-        "log_Fmort"
-      ))
-    }
+    # if (!"index" %in% fleet_types) {
+    #   module_fields <- setdiff(module_fields, c(
+    #     "log_q",
```

Comment:
might want to consider a logit option for q to force q to be below some value. Scale is difficult to estimate sometimes.